fishermen/cmd/run-application-checks/base.go

```diff
@@ -94,9 +97,13 @@ func RunApplicationChecks(ctx context.Context, requestID string, performChecks f
 	if len(redisConnectionStrings) <= 0 {
 		return shared.ErrNoCacheClientProvided
 	}
-	db, err := database.ClientFromURI(ctx, mongoConnectionString, mongoDatabase)
+	dbClient, err := database.NewPHDClient(dbclient.Config{
+		BaseURL: phdBaseURL,
+		APIKey:  phdAPIKey,
+		Version: dbclient.V0,
```

Comment:
I suggest we change this to use V1. V1 endpoints are confirmed to work by multiple other repos already running in production and if we go right to V1 we don't need another PR. Aiming to remove V0 from PHD next week.